File: C,C++/p10921.c
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

#include <stdio.h>
#include <string.h>

int main () {
    char data[100];
    while(scanf("%s", data) != EOF) {
        int i;
        for (i = 0; i < 100; i++) {
            switch (data[i]) {
                case 'A' :
                    printf("2");
                    break;
                case 'B' :
                    printf("2");
                    break;
                case 'C' :
                    printf("2");
                    break;
                case 'D' :
                    printf("3");
                    break;
                case 'E' :
                    printf("3");
                    break;
                case 'F' :
                    printf("3");
                    break;
                case 'G' :
                    printf("4");
                    break;
                case 'H' :
                    printf("4");
                    break;
                case 'I' :
                    printf("4");
                    break;
                case 'J' :
                    printf("5");
                    break;
                case 'K' :
                    printf("5");
                    break;
                case 'L' :
                    printf("5");
                    break;
                case 'M' :
                    printf("6");
                    break;
                case 'N' :
                    printf("6");
                    break;
                case 'O' :
                    printf("6");
                    break;
                case 'P' :
                    printf("7");
                    break;
                case 'Q' :
                    printf("7");
                    break;
                case 'R' :
                    printf("7");
                    break;
                case 'S' :
                    printf("7");
                    break;
                case 'T' :
                    printf("8");
                    break;
                case 'U' :
                    printf("8");
                    break;
                case 'V' :
                    printf("8");
                    break;
                case 'W' :
                    printf("9");
                    break;
                case 'X' :
                    printf("9");
                    break;
                case 'Y' :
                    printf("9");
                    break;
                case 'Z' :
                    printf("9");
                    break;
            	case '1' :
                    printf("1");
                    break;
		case '0' :
                    printf("0");
                    break;
		case '-' :
                    printf("-");
                    break;
                default :
                    break;
            }
        }
        printf("\n");
        
        for (i = 0; i < 100; i++) {
            data[i] = '\0';
        }
    }
    
    return 0;
}

File: Java/p10921.java

import java.util.Scanner;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author DarKnight98
 */
public class p10921 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        while(input.hasNext()) {
            String res = "";
            String masukan = input.nextLine();
            for (int i = 0; i < masukan.length(); i++) {
                switch (masukan.charAt(i)) {
                    case 'A' :
                        res += "2";
                        break;
                    case 'B':
                        res += "2";
                        break;
                    case 'C' :
                        res += "2";
                        break;
                    case 'D':
                        res += "3";
                        break;
                    case 'E' :
                        res += "3";
                        break;
                    case 'F' :
                        res += "3";
                        break;
                    case 'G' :
                        res += "4";
                        break;
                    case 'H' :
                        res += "4";
                        break;
                    case 'I' :
                        res += "4";
                        break;
                    case 'J' :
                        res += "5";
                        break;
                    case 'K' :
                        res += "5";
                        break;
                    case 'L' :
                        res += "5";
                        break;
                    case 'N' :
                        res += "6";
                        break;
                    case 'M' :
                        res += "6";
                        break;
                    case 'P' :
                        res += "7";
                        break;
                    case 'Q' :
                        res += "7";
                        break;
                    case 'R':
                        res += "7";
                        break;
                    case 'S' :
                        res += "7";
                        break;
                    case 'T' :
                        res += "8";
                        break;
                    case 'U':
                        res += "8";
                        break;
                    case 'V' :
                        res += "8";
                        break;
                    case 'W':
                        res += "9";
                        break;
                    case 'X':
                        res += "9";
                        break;
                    case 'Y':
                        res += "9";
                        break;
                    case 'Z' :
                        res += "9";
                        break;
                    case '-' :
                        res += "-";
                        break;
                    case '1' :
                        res += "1";
                        break;
                    case '0' :
                        res += "0";
                        break;
                    default:
                        res += "6";
                        break;
                }
            }
            
            System.out.println(res);
        }
    }
}
